wallet/how-to/manage-permissions.md

```diff
@@ -32,7 +32,7 @@ the [`eth_accounts`](/wallet/reference/eth_accounts) restricted method:
 To access accounts, we recommend using [`eth_requestAccounts`](/wallet/reference/eth_requestAccounts),
 which automatically asks for permission to use `eth_accounts` by calling `wallet_requestPermissions`
 internally.
-See [how to access a user's accounts](connect/access-accounts.md) for more information.
+See [how to access a user's accounts](access-accounts.md) for more information.
```

Comment:
Usually, "for more information" is implied in a xref, and supported by the name of the content being referenced.